### ISSUE TITLE
Flareact Consideration

### ISSUE THREAD

**Issue by abdelhamid-attaby** [fab-spec/fab#368]
First of all, thanks @geelen for this great project, it is really a game changer and I believe it will get a huge impact in the future.

I would like to suggest considering converting [Flareact](https://flareact.com/) into FAB as it is a very minified SSR-support React framework and well tested in Cloudflare Workers.

The bundle size produced is around 100 KB for an initial project with a milliseconds response time.

**Comment by geelen** [comment]
Yes! I haven't looked into the internals of Flareact, but I'd be super keen to see how easy it is to convert. It should just be a matter of figuring out what needs to go where, and wiring up its server side renderer to the FAB middleware. Is that something you'd be able to help with? I'll have some time this week to assist but otherwise my focus is on NextJS just at the moment.

**Comment by abdelhamid-attaby** [comment]
@geelen they are mentioning that it is almost the same as `Next.js` APIs 

> Flareact is modeled after the terrific Next.js project and its APIs. If you're transitioning from Next.js, a lot of the APIs will seem familiar, if not identical!

I also inspected the assets generated from the build against the ones generated from `Next.js` and they are almost similar.  The only challenging now for me is the `workers` file which should be mapped to the `server.js` file. 

I can start creating an `input-flareact` package and ask for your help if I need.
